apps/google-analytics-4/frontend/src/components/main-app/ChartContent/ChartContent.styles.ts

```diff
@@ -0,0 +1,11 @@
+import { css } from 'emotion';
+
+export const styles = {
+  root: css({
+    width: '100%',
+    height: '10rem',
```

Comment:
where is this magic number coming from?

Reply:
sure, this number is derived from the height of the current chart component in the app given the space. Adding this constraint here also made the container height consistent so when the warning message appears instead of the chart, it is centered and does not shift around the contents. Everything maintains the 10rem. Happy to add a note within the code here?